### ISSUE TITLE
BioModel Discussion Issue for eos1n4b

### ISSUE THREAD

**Issue by Zainab-ik** [ersilia-os/eos1n4b#8]
**Summary !!!**

- **BioModel Name**: Li2021 - HDAC3i-Finder: A Machine Learning-based Computational Tool to Screen for HDAC3 Inhibitors.

- **BioModel Tag**: Machine learning model, Ersilia, FAIR AIML

- **Metadata Comments**: 
The model predicts the inhibitory potential of small molecules against Histone deacetylase 3 (HDAC3)

1. The drug target "Histone deacetylase 3" is related to different diseases such as cancer, and diabetes. Those aren't related metadata. 
2. This model was built using 5 algorithm and 3 descriptors; 
Algorithm - k-Nearest Neighbour (KNN), Support Vector Machine (SVM), Random forest (RF), eXtreme Gradient Boosting (XGBoost), Deep Neural Network (DNN).
Descriptors - Mordred descriptors, MACCS key, Morgan fingerprint. 
3. The best performing model is the XGBoost with the Morgan fingerprint. (that's the deployed model to the GUI application)
For our annotation, we'd only be including the best performing model and its feature.
4. We have an _ROC enrichment_ as an evaluation metrics between the validation and training dataset. I'm not sure it fits into the metadata. What do you think?

- **Contributor**:
> 
 | roles             | Names | 
 | :---------------- | :------: | 
 | Curator        |   @Zainab-ik   | 
 | Code contributor      |   @Richiio     | 
 | Reviewer       |   @GemmaTuron    | 

- **Curation status** :  manually curated

- **Annotation file**: [here](https://docs.google.com/spreadsheets/d/1zf4w9cBboKpXtBQMHpIVCVHVorEsSuXvNZxnYW_0WX8/edit?usp=sharing)

**Comment by Zainab-ik** [comment]
Hi @GemmaTuron 
Annotation complete. Kindly review.

**Comment by Zainab-ik** [comment]
This model was uploaded during the hackathon. I've currently reviewed, and updated the submission.
